templates/html/.filters/all.js

```diff
@@ -117,8 +117,9 @@ module.exports = ({ Nunjucks, Markdown, OpenAPISampler }) => {
   });
 
   Nunjucks.addFilter('getPayloadExamples', (msg) => {
-    if (Array.isArray(msg.examples()) && msg.find(e => e.payload)) {
-      return msg.filter(e => e.payload);
+    if (Array.isArray(msg.examples()) && msg.examples().find(e => e.payload)) {
+      // Instead of flat or flatmap use this.
+      return msg.examples().map(e => e.payload).filter(Boolean).reduce((acc, val) => acc.concat(val), []);
```

Comment:
That's fine to me but you have lodash in the context, so you could use `_.flatMap`: https://lodash.com/docs/4.17.15#flatMap.

Just add `_` here: https://github.com/asyncapi/generator/pull/165/files#diff-01dfe915256af0d2b6dc5d1ea3de4dedL1, like this:

```js
module.exports = ({ _, Nunjucks, Markdown, OpenAPISampler }) => {
```

Reply:
Even better :+1: